autocomplete/definitions/global/table/filter.lua

```diff
@@ -8,7 +8,7 @@ Any additional arguments will be passed to `f`. For example, `table.filter(t, f,
 ]],
 	arguments = {
 		{ name = "t", type = "table" },
-		{ name = "f", type = "fun(k: unknown, v: unknown, ...)", "The function to use when filtering values of `t`. (This is sometimes called a predicate function.)" },
+		{ name = "f", type = "fun(k: unknown, v: unknown, ...): boolean|unknown", "The function to use when filtering values of `t`. (This is sometimes called a predicate function.)" },
```

Comment:
It's not enforced in the implementation, but let's limit the return value to `boolean`.

Reply:
good idea